Remove Fedora-specific defaults from Noggin

Conan-Kudo opened the pull request:
This makes it easier for third-party (non-Fedora) deployments of Noggin to work.

abompard commented:
Well, I would have set the news type as `bic` (backwards incompatible), because it's a default configuration change.

abompard reviewed (changes requested):
The news type should be bic (backwards incompatible), because it's a default configuration change.

Also please allow changes to your branch so that mergify can rebase. Thanks!

Conan-Kudo committed:
Remove Fedora-specific defaults from Noggin

This makes it easier for third-party (non-Fedora) deployments of Noggin to work.

Signed-off-by: Neal Gompa <neal@gompa.dev>

Conan-Kudo commented:
@abompard Done.
